src/components/Calendar.tsx

```diff
@@ -87,12 +91,16 @@ function getViewPeriodStart(
   if (view === 'day') {
     return startOfDay(referenceDate);
   }
+  if (view === 'custom') {
```

Comment:
Maybe this could be part of the if statement above as the return value is the same? The if isn't really necessary in a TypeScript world, but I want to throw this error below if people are using JS.

Reply:
fixed